### PR TITLE
feat: Created a compare function in compare branch

### PR DESCRIPTION
**Pull request by pr-th-m**
Request you guys to review my code, and report mistakes if any.

### CODE REVIEW

File: array.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+ 
+//Rest parameter
```
Comment (KrithikaSuvarna01) [inline array.js:2]: Can you please provide Java Script Documentation to this function!

**Review by KrithikaSuvarna01 — COMMENTED** [review]
Can you please add some JS docs to it

File: array.js (hunk 1 above)
Comment (pr-th-m) [reply array.js:2]: Resolved

**Review by KrithikaSuvarna01 — APPROVED** [review]
LGTM..

**Review by hrithik-b01 — APPROVED** [review]
**LGTM**

**Review by NIREE23 — COMMENTED** [review]
Good work

**Review by NIREE23 — APPROVED** [review]

**Pull request merged**